modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -222,13 +222,11 @@ components:
                 externalReferenceCode:
                     description:
                         The user group's external reference code.
-                    readOnly: true
                     type: string
                 id:
                     description:
                         The user group's ID.
                     format: int64
-                    readOnly: true
```

Comment:
@adolfopa, is this necessary? Normally ID is always readOnly

Reply:
I agree with @jkappler, why did you remove this? Even more in this case. We are not updating the User Group itself but the relation with the Space.

Reply:
This can be removed, as we no longer have POST over those resources. I mentioned this in our Slack channel two days ago (request data was an empty object when deserializing).